lib/project-config.ts

```diff
@@ -8,49 +8,70 @@ export type projectInfo = {
     urlPrefix: string; // url to 'listserv' of mail (should it be in mailrepo?)
 };
 
+export interface IRepoConfig {
+    name: string; // name of the repo
+    owner: string; // owner of repo holding the notes (tracking data)
+    baseOwner: string; // owner of upstream ("base") repo
```

Comment:
@webstech since I started referring to the corresponding repository as `upstream-repo`, what would you think about the idea to rename this from `baseOwner` to `upstreamOwner`?

Reply:
I have no issue with renaming.  _My_ choice in names may not always be as git oriented as they could be.

Reply:
For the record, I am going forward with this: https://github.com/dscho/gitgitgadget/commit/2de5309eea4d9c1dfa387e9e69eec58c3aa24db0